packages/v4/src/content/design-guidelines/components/card/card.md

```diff
@@ -7,112 +7,79 @@ A **card** is a square or rectangular container that can contain any kind of con
 
 ## Elements
 
-A card usually consists of 3 parts: 
+A card usually consists of 3 parts:
 
 <img src="./img/card.png" alt="3 parts of a card" width="394"/>  
 
 
 1. **Header**: Title of the information group
 2. **Body**: Brief details about the group
-3. **Footer**: Links to access more information about the group 
+3. **Footer**: Links to access more information about the group
```

Comment:
Just realized that we say the footer is for links, yet the example picture doesn't have a link haha

Reply:
I will revise these examples. Agree that the current information is confusing.